### DISCUSSION TITLE
Nightingale 2.0: how to add back a deleted endpoint?

### DISCUSSION THREAD

**Discussion by czxin788** [ccfos/nightingale#1889] [General]
夜莺2.0版本，删除的endpoint，用原来的名字加不会来了，只能改主机名才行。

![image](https://github.com/ccfos/nightingale/assets/13861904/3abc961e-c408-46ee-839e-316399eb361e)

**Comment by UlricQin** [comment]
太久了，已经忘记了。看现象，像是在某个地方有缓存，可以重启一下夜莺的进程试试。
